da_playground_install/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Install or Update Docassemble package
+description: |
+    To continually deploy docassemble packages, this action will install GitHub packages to a given docassemble server's playground.
+
+    Will install the package from the currently checked out repository (i.e. you must run `actions/checkout` in your own action first).
+inputs:
+  SERVER_URL:
+    description: "The URL of the docassemble server that you want to install this package on"
+    required: true
+  DOCASSEMBLE_DEVELOPER_API_KEY:
+    description: "API key of an account that has premissions to install packages to the docassemble server"
+    required: true
+  PROJECT_NAME:
+    description: 'Name of the playground project to push to'
+    required: true
+  USER_ID:
+    description: User ID for the Docassemble user (defaults to the account the API key came from)
```

Comment:
Is this an ID or an email?

Reply:
Going to leave it in, even if it's a bit cumbersome to use. We match the docassemble API for the rest of the params, would be weird to leave one out.